scRNA-seq/01-scRNA_quant_qc.Rmd

```diff
@@ -245,6 +245,19 @@ Look for the `10X_P4_3_subset-qc_report.html` file created in the `qc-reports` d
 
 We have also placed an example of a poor quality sample alevinQC report in the `qc-reports` directory, with the name `Bad_Example_10X_P4_2_qc_report.html`.
 
+This report will show a few key metrics that inform you about the quality of your sample. 
+There is a lot of information included in the report, so some key metrics to note are included in the `Summary tables`:
+
+- Fraction of reads in whitelist barcodes 
+- Mean number of reads per cell
+- Median number of detected genes per cell 
+
+The fraction of reads in whitelist barcodes is particularly important as a low percentage here means the library contains reads that do not correspond to cell barcodes, indicating poor single-cell capture. 
+The mean number of reads per cell and median number of detected genes per cell can be helpful in understanding how deep the library was sequenced, as the higher these numbers are, the more information you will obtain per cell. 
+
+The `Knee plot` shows the frequency of each barcode being detected and the estimation of the `knee` cutoff. 
```

Comment:
> I think the knee plot is often very confusing, so it is probably worth explaining what the two axes are here, and maybe even what the "knee" is.

I updated the knee plot description. I was struggling a bit because of how much detail to go into without being overly confusing. I do think that this will be a big point of discussion during teaching as it is confusing for people. Let me know what you think.